### PR TITLE
[pharmacy] Change visual layout for drug details

### PR DESCRIPTION
**Pull request by seshness**
Summary:
Screenshot: http://d.pr/i/uMcm

This needs more styling work, but here's an initial effort.
I'm planning on getting rid of the dashed box when the time graph comes along.

### CODE REVIEW

File: app/views/pharmacy/index.html.haml [hunk 1]
```diff
@@ -22,23 +22,29 @@
...
-                  :class => 'alert_field'
+                :class => 'alert_field', :placeholder => drug.units
                 = submit_tag 'Update', :class => 'submit'
             %div.details_right
-              %div.bigSparkline{:id => "bigSparkline-#{drug.id}"}
-                Sparkline unavailable
+              Estimated rate:
+              = drug.estimated_rate * 1.week
+              = drug.units
+              per week
```
Comment (danwang) [inline app/views/pharmacy/index.html.haml]: ??? Duplicate of line above?

File: app/views/pharmacy/index.html.haml [hunk 2]
```diff
@@ -22,23 +22,29 @@
...
+              = form_tag pharmacy_drug_path(drug.id),
+                :class => :drug_form,
```
Comment (danwang) [inline app/views/pharmacy/index.html.haml]: How to differentiate between the two forms? Can we use IDs or different classes?

**Comment by danwang** [comment]
I moved the tables to a partial for better abstraction. I also fixed the problem with the slideToggle being jumpy without an absolute height. The problem was that originally, all `div`s inside the hidden element were being animated. This causes the height of the element to change non-linearly. I just added a class to the outermost div and animated that one instead.

Also added `width: 50%` to prevent the big sparkline from falling down. Not sure why this wasn't there earlier.

Also removed useless IDs (multiple elements had the same ID). The new IDs are on labels; it might be good to put IDs on the forms as well.

**Comment by danwang** [comment]
The borders were really janky. The solution is not to have partial borders on different elements, because once you add more, you end up with lots of problems. I just added a border to the `tbody` and set `border-collapse: none`.

Also set `float:left` on all elements, because it doesn't matter.

**Comment by seshness** [comment]
Screenshot please?

Edit: never mind, http://d.pr/i/Kcwg is representative

**Comment by seshness** [comment]
@danwang: The point of the previous commit was to move things around in order to give the time graph some more room to breathe. The really interesting part of the larger graph is the horizontal time axis, whose range may consist of anything between days and years.

Please be careful when pushing your commits to a pull request branch made by someone else - the placeholder values didn't get copied resulting in some duplicate work now.

Thanks for cleaning up the files.

**Comment by danwang** [comment]
Sorry about that. The ajax calls got broken. I fixed them and made them a little more resistant to DOM changes.

**Comment by seshness** [comment]
Ideally the alert feature would be in a different pull request..but it looks okay to me. Do the cucumber tests pass?

**Comment by seshness** [comment]
The cucumber tests didn't pass ^.

I'll fix them and put the fix in a separate pull request.

@danwang: I'm taking your commits on top of this as an implicit okay-to-merge, and will proceed shortly.

**Pull request merged**